Source/Diagnostics/FlushFormats/FlushFormat.H

```diff
@@ -5,6 +5,7 @@
 
 #include "Diagnostics/ParticleDiag/ParticleDiag.H"
 #include "Particles/MultiParticleContainer.H"
+#include "WarpX.H"
```

Comment:
Is this needed for something referenced in the header? I don't see anything added. This can potentially cause rebuilds for lots of irrelevant changes.

Reply:
This change is from Andrew's PR, so it would be better to just make the same comment on his upstream PR.